tests/run_all_tests.sh

```diff
@@ -43,33 +43,41 @@ handle_errors () {
 }
 
 # Run battery of core FLAX API tests.
-pytest tests -n auto $PYTEST_OPTS
-
-# validate types
-if [[ "$OSTYPE" == "darwin"* ]]; then
-  echo "Pytype is currently not working on MacOS, see https://github.com/google/pytype/issues/661"
-else
-  pytype flax/
-fi
+pytest -n 4 tests $PYTEST_OPTS
 
 # Per-example tests.
-if [[ $ALL_EXAMPLES == 'true' ]]; then
-  # we apply pytest within each example to avoid pytest's annoying test-filename collision.
-  # In pytest foo/bar/baz_test.py and baz/bleep/baz_test.py will collide and error out when
-  # /foo/bar and /baz/bleep aren't set up as packages.
-  for egd in $(find examples -maxdepth 1 -mindepth 1 -type d); do
-      pytest $egd
-  done
-fi
-
-# Per-example tests for Linen examples.
-for egd in $(find linen_examples -maxdepth 1 -mindepth 1 -type d); do
+#
+# we apply pytest within each example to avoid pytest's annoying test-filename collision.
+# In pytest foo/bar/baz_test.py and baz/bleep/baz_test.py will collide and error out when
+# /foo/bar and /baz/bleep aren't set up as packages.
+for egd in $(find examples -maxdepth 1 -mindepth 1 -type d); do
     pytest $egd
+
     # use cd to make sure pytpe cache lives in example dir and doesn't name clash
     # use *.py to avoid importing configs as a top-level import which leads tot import errors
     # because config files use relative imports (e.g. from config import ...). 
     (cd $egd ; pytype "*.py")
```

Comment:
Do you want to test pytype twice for examples ?
(see line 78 below)

Reply:
Oops, I was missing a commit that I forgot to push.